apm-lambda-extension/extension/logger.go

```diff
@@ -31,3 +35,29 @@ func init() {
 	newLoggerWithFields := newLogger.WithFields(logrus.Fields{"event.dataset": "apm-lambda-extension"})
 	Log = newLoggerWithFields
 }
+
+// ParseLogLevel parses s as a logrus log level.
+func ParseLogLevel(s string) (logrus.Level, error) {
+	// Set default output. Required to start logging back if the prior level was "off"
+	logrus.SetOutput(os.Stderr)
+	switch strings.ToLower(s) {
+	case "trace":
+		return logrus.TraceLevel, nil
+	case "debug":
+		return logrus.DebugLevel, nil
+	case "info":
+		return logrus.InfoLevel, nil
+	case "warn", "warning":
+		// "warn" exists for backwards compatibility;
+		// "warning" is the canonical level name.
+		return logrus.WarnLevel, nil
+	case "error":
+		return logrus.ErrorLevel, nil
+	case "fatal", "critical":
+		return logrus.FatalLevel, nil
+	case "off":
```

Comment:
Add `panic` again to support default logrus functionality

Reply:
I'd suggest only using the log levels that are supported from the agent logging spec. This should give enough flexibility and allows `ELASTIC_APM_LOG_LEVEL` to be picked up by both the agent and the extension.

Reply:
Ok, thanks for the suggestion, let us stick to the agent logging spec then. I was thinking that we could easily expose Logrus functionality aside from the spec-defined levels, but it is not of great use if `panic` is not supported by the agent...